app/pages/project/project-page.jsx

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import Translate from 'react-translate-component';
+import { Markdown } from 'markdownz';
+import { sugarClient } from 'panoptes-client/lib/sugar';
+import ProjectNavbar from './project-navbar';
+import PotentialFieldGuide from './potential-field-guide';
+
+export default class ProjectPage extends React.Component {
+  constructor() {
+    super();
+    const lastSugarSubscribedID = null;
+  }
+
+  getChildContext() {
+    return this.context.geordi;
+  }
+
+  componentDidMount() {
+    document.documentElement.classList.add('on-project-page');
+    this.updateSugarSubscription(this.props.project);
+    (this.context.geordi) ? this.context.geordi.remember({ projectToken: this.props.project.slug }) : null;
+  }
```

Comment:
`this.context.geordi && this.geordi.remember(…)` would work here.

Reply:
`d'oh! Should have been `this.context.geordi && this.context.geordi.remember(…)`

Reply:
I figured 👍